.github/workflows/c-cpp.yml

```diff
@@ -2,7 +2,7 @@ name: C/C++ CI
 
 on:
   push:
-    branches: [master, develop, testing]
+    branches: [master, develop, testing, feature/libusb_cmake]
```

Comment:
We don't need this here. The workflow is called upon merging into testing and the `testing` branch may break, that's totally acceptable for it's dedicated purpose. ;-)

Reply:
Yes, Correct -I forgot to remove it, Will update ASAP!